clubhouse/__init__.py

```diff
@@ -1,60 +1,3 @@
 __version__ = "0.3.0"
 
-import logging
-from os import path
-from typing import Dict
-from urllib.parse import urlparse
-
-import requests
-
-ENDPOINT_HOST = "https://api.clubhouse.io"
-ENDPOINT_PATH = "/api/v3"
-
-ClubhouseStory = Dict[str, object]
-ClubhouseUser = Dict[str, object]
-ClubhouseComment = Dict[str, str]
-ClubhouseTask = Dict[str, str]
-ClubhouseLabel = Dict[str, str]
-ClubhouseFile = Dict[str, str]
-
-logger = logging.getLogger("clubhouse")
-
-
-class ClubhouseClient(object):
-    def __init__(self, api_key, ignored_status_codes=None):
-        self.ignored_status_codes = ignored_status_codes or []
-        self.api_key = api_key
-
-    def get(self, *segments, **kwargs):
-        return self._request("get", *segments, **kwargs)
-
-    def post(self, *segments, **kwargs):
-        return self._request("post", *segments, **kwargs)
-
-    def put(self, *segments, **kwargs):
-        return self._request("put", *segments, **kwargs)
-
-    def delete(self, *segments, **kwargs):
-        return self._request("delete", *segments, **kwargs)
```

Comment:
I think there is some value keeping those around as an escape hatch and for backward compatibility.

Reply:
You're right, I'll add those back in. Thank you.